selfdrive/ui/qt/widgets/cameraview.cc

```diff
@@ -253,7 +253,6 @@ void CameraViewWidget::vipcConnected(VisionIpcClient *vipc_client) {
   stream_width = vipc_client->buffers[0].width;
   stream_height = vipc_client->buffers[0].height;
 
-  glPixelStorei(GL_UNPACK_ALIGNMENT, 1);
```

Comment:
@deanlee this breaks text rendering in the OpenGL widget occasionally. Everything seems to work find without changing the alignment. Do you remember why this was added?